chainntnfs/btcdnotify/btcd.go

```diff
@@ -711,13 +587,7 @@ func (b *BtcdNotifier) RegisterSpendNtfn(outpoint *wire.OutPoint,
 // confirmationNotification represents a client's intent to receive a
 // notification once the target txid reaches numConfirmations confirmations.
 type confirmationsNotification struct {
-	txid *chainhash.Hash
-
-	initialConfirmHeight uint32
-	numConfirmations     uint32
-
-	finConf      chan *chainntnfs.TxConfirmation
-	negativeConf chan int32 // TODO(roasbeef): re-org funny business
+	chainntnfs.ConfNtfn
```

Comment:
Seems like there's no need for an additional type in this case? As is, it'll just replicate the existing struct with a diff name.

Reply:
That is right, but I kept it for symmetry with neutrinonotify. I don't believe there's any overhead to having this wrapper struct.